lix/packages/client/src/git/checkout.ts

```diff
@@ -68,7 +86,7 @@ export async function checkout({
 	force = false,
 	track = true,
 	cache = {},
-}) {
+}: any) {
```

Comment:
the type annotations do not work anymore, if you know how to activate them in ts pls let me know. i think using a big plugin or build step is too much hassle, then we just convert it next time we touch this. as doCheckout is not exposed as lix api this is probably fine until then

Reply:
Yeah - did some research https://alexharri.com/blog/jsdoc-as-an-alternative-typescript-syntax#:~:text=JSDoc%20does%20not%20work%20in,frustrating%2C%20especially%20for%20larger%20modules. :-( we can define the types later on for now i agree any should do the trikc